### PR TITLE
[TASK][27007] layout application

### PR DESCRIPTION
**Pull request by chinhhq-0603**
https://edu-redmine.sun-asterisk.vn/issues/27007

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<title><%= full_title yield :title %></title>
+<footer class="footer">
+  <small>
+    <%= t(".content1_html", href: link_to(t(".href_title1"), Settings.links.rail_tutorial_link)) %>
+    <%= t(".content2_html", href: link_to(t(".href_title2"), Settings.links.rail_tutorial_link)) %>
```
Comment (matTroiCuaHieu) [inline app/views/layouts/_footer.html.erb:5]: bỏ ngoặc

**Review by matTroiCuaHieu — APPROVED** [review]

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (chinhhq-0603) [reply app/views/layouts/_footer.html.erb:5]: ok a

**Comment by chinhhq-0603** [comment]
ready

File: app/views/shared/_error_messages.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<% if object.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t ".error_count_text" %> <%= pluralize object.errors.count, t(".error") %>
```
Comment (hieplt-0914) [inline app/views/shared/_error_messages.html.erb:4]: size

**Pull request merged**